### ISSUE TITLE
Alignment through format string needed when printing line numbers in Find result page

### ISSUE THREAD

**Issue by jxramos** [notepad-plus-plus/notepad-plus-plus#1647]
When searching a file for a string I noticed that some of the results were misaligned which threw me in using alignment to ascertain differences in similar regions of code. It would be good if the Find results page mirror the formatting done correctly in the line numbers of the editor itself.

![searchlinealignment](https://cloud.githubusercontent.com/assets/13857038/13867226/39014ee2-ec7a-11e5-849b-f2de97466d90.png)

If you look at the line numbers in the editor you can see they format correctly as 

```
 87
 89
...
 98
 99
100
```

**Comment by MetaChuh** [comment]
we are closing this issue due to multiple newer notepad++ versions since it was filed.

please update your notepad++ to the latest version [<kbd>Notepad++ 7.6.6</kbd>](https://notepad-plus-plus.org/download/v7.6.6.html) and migrate or reinstall your plugins.
if your issue still persists, and it is still of relevance to you, please feel free to file a new issue as detailed and reproducible as possible.

thank you for your help and best regards.

-----
```
general notes:

please close your issues as soon as you are satisfied, to help
keeping the notepad++ issue tracker clean.

when creating a new issue, please provide all information as shown at the issue template.
your notepad++ debug info is part of the minimum requirements.
(? > Debug Info... > Copy debug info into clipboard).

please also conduct a search at the issue tracker beforehand, to check if a similar issue
already exists.

for general questions, or if it is not sure, whether your issue is directly related to
the notepad++ source code, please visit us at the notepad++ community forum, and
search if related topics exists. you are welcome to post either at similar
topics, or to create a new topic at Help Wanted or General Discussion.
```
[<p align="center"><kbd> &emsp; &emsp; &emsp; &emsp; &emsp; &emsp; &emsp; &emsp; &emsp; &emsp; &emsp; &emsp; Click here to visit the Notepad++ Community Forum &emsp; &emsp; &emsp; &emsp; &emsp; &emsp;&emsp; &emsp; &emsp; &emsp; &emsp; &emsp; &emsp; </kbd></p>](https://notepad-plus-plus.org/community/)

**Comment by jxramos** [comment]
Turns out this issue was never addressed as of v7.7

![image](https://user-images.githubusercontent.com/13857038/58992232-139b3780-879f-11e9-829b-52946e553868.png)
